src/boards/rainbow_esp.cpp

```diff
@@ -75,6 +75,8 @@ std::vector<uint8> readHostFile(std::string const& file_path) {
 
 }
 
+std::array<std::string, NUM_FILE_PATHS> dir_names = { "SAVE", "ROMS", "USER" };
```

Comment:
Pu it in the anonymous namespace (just above). Il will make it local to this file and avoid linking error if another .cpp declare a variable named `dir_names`